lib/class-wp-theme-json-gutenberg.php

```diff
@@ -1421,6 +1428,7 @@ public function get_custom_css() {
 	 * @return string The global styles base custom CSS.
 	 */
 	public function get_base_custom_css() {
```

Comment:
So these never made it to Core, right? Should we remove the `@since` annotations and maybe add a "don't backport note"?

Doesn't have to be here, just wondering if it'll help later.

Reply:
Oh good point, we should remove the `@since` on these!